lib/server.js

```diff
@@ -4,10 +4,10 @@ const die = e => { console.error(e); process.exit(1) }
 const { getUpstreamRegistry } = require('./upstream')
 const app = express().disable('x-powered-by').set("json spaces", 4)
 const server = http.createServer(app)
-const { description, version } = require('../package.json')
+const { "registries-server": config, description, version } = require('../package.json')
 
 const PORT = Number(process.env.PORT || 80)
-const REGISTRY_REPOSITORY = process.env.REGISTRY_REPOSITORY || "https://github.com/ISDCF/registries"
+const REGISTRY_REPOSITORY = process.env.REGISTRY_REPOSITORY || config["registry-repository"]
```

Comment:
Shouldn't a default still exist, or the `registry-server/registry-repository` field should be mandatory.

Reply:
> Shouldn't a default still exist

I considered it, but if there is a concern that someone  might muck with `package.json` then there should be an equal concern that they could muck w/, say, `dependencies`, or `scripts`. Either of which would break functionality. 

To me, we've got the sensible default in `package.json`. There should be no inference that that setting is somehow optional.

Reply:
Ok. Can the server die explicitly and gracefully if the registry is available in neither package.json nor an env variable?